README.md

```diff
@@ -349,24 +379,20 @@ the ticket to indicate that you are working on it. Don't hesitate to ask
 clarifying questions on the ticket as needed, too, if anything is unclear.
 
 #### Submitting Code
-
 For code contributions, please open up a pull request! PRs will get OSPR tickets
 assigned to them, as mentioned in the above contributing guidelines.
 
 #### Approval by UX and Front End Team Members
-
 An edX UX Team member will be working with you on any pull requests you make.
 They be evaulating your pull request from a design point of view as well as from
```

Comment:
Just noticed that this is missing a "will" - "They will be evaluating..." (Unless we really are that hip.)